src/olympia/reviewers/serializers.py

```diff
@@ -95,9 +96,11 @@ def _get_hash_for_selected_file(self, obj):
         commit = self._get_commit(obj)
         tree = self.repo.get_root_tree(commit)
 
-        cache_key = (
+        cache_key = make_key(
             f'reviewers:fileentriesserializer:hashes'
-            f':{commit.hex}:{selected_file}')
+            f':{commit.hex}:{selected_file}',
+            with_locale=False,
+            normalize=True)
```

Comment:
I don't disagree with the solution, but wouldn't it have been enough to normalize/encode `selected_file`?

Reply:
I think that makes sense and ensures the key stays readable. 
0ce5268

Reply:
As @diox correctly pointed out we need to avoid to exceed memcached max key lengh so `make_key` over the whole cache key makes much more sense. I added a comment.